prisma/migrations/20211115182559_availability_issue/migration.sql

```diff
@@ -0,0 +1,21 @@
+
+insert into "Availability" ("userId", "startTime", "endTime", "days")
+select
+  id as "userId", 
+  CAST(CONCAT(CAST(("startTime" / 60) AS text), ':00') AS time) AT TIME ZONE "timeZone" AT TIME ZONE 'UTC',
+  CAST(CONCAT(CAST(("endTime" / 60) AS text), ':00') AS time) AT TIME ZONE "timeZone" AT TIME ZONE 'UTC',
```

Comment:
1. takes `startTime` (minutes from midnight)
2. converts it to a `TIME`
3. converts it from timezone `user.timeZone` --> `UTC`